packages/expo-barcode-scanner/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### 💡 Others
 
+- `BarCodeScannerResult` now returns an additional `raw` field corresponding to the barcode value as it was encoded in the barcode without parsing. Will always be undefined on iOS.
```

Comment:
I think it makes more sense to move this a few lines up to new features section 😉